src/realm/object-store/shared_realm.hpp

```diff
@@ -378,6 +378,26 @@ class Realm : public std::enable_shared_from_this<Realm> {
         return !m_transaction && !m_coordinator;
     }
 
+    /**
+     * Deletes the following files for the given `realm_file_path` if they exist:
+     * - the Realm file itself
+     * - the .management folder
+     * - the .note file
+     * - the .log file and its legacy versions: .log_a and .log_b
+     * - the backup file
```

Comment:
Is this correct? The backup file is not on the file list, is it?

Reply:
You are of course absolutely right. We've changed that back in the `DB` class and removed `backup` again after discussing about it. This comment was not updated after that.
Should be fixed now.